_posts/2020-07-28-accelerating-training-on-nvidia-gpus-with-pytorch-automatic-mixed-precision.md

```diff
@@ -104,13 +104,14 @@ Training accuracy: NVIDIA DGX-1 (8x V100 16GB)
   </tbody>
 </table>
 
-## Speedup Performance: 
+## Speedup Performance:
 
 ### FP16 on NVIDIA V100 vs. FP32 on V100
 AMP with FP16 is the most performant option for DL training on the V100. In Table 1, we can observe that for various models, AMP on V100 provides a speedup of 1.5x to 5.5x over FP32 on V100 while converging to the same final accuracy.
 
 <div class="text-center">
-  <img src="{{ site.url }}/assets/images/nvidiafp32onv100.jpg" width="100%">
+  <img class="blog-image" data-toggle="modal" data-target="#blog-modal" src="{{ site.url }}/assets/images/nvidiafp32onv100.jpg" width="100%">
+  {% include blog_modal.html blog-image="nvidiafp32onv100.jpg"%}
```

Comment:
Is it possible to move `<img class="blog-image" data-toggle="modal" data-target="#blog-modal" src="{{ site.url }}/assets/images/nvidiafp32onv100.jpg" width="100%">` into the partial? Then all they'd have to do is add `{% include blog_modal.html blog-image="nvidiafp32onv100.jpg"%}` instead of the img tag with all of that data stuff.